src/doc/using_visit/VisualizationWindows/Interactor_settings.rst

```diff
@@ -21,47 +18,27 @@ that have interactor properties. The **Interactors window** is shown in
 Zoom interactor settings
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-The zoom interactor settings are mostly used when the vis window is in
-zoom mode. When the vis window is in zoom mode, clicking in the vis window
-will anchor a point that becomes one of the corners of a zoom rectangle.
-When you release the mouse, the point over which the mouse was released
-becomes the opposite corner of the zoom rectangle. VisIt's default behavior
-is to show guidelines that extend from the edges of the zoom rectangle to
-the edges of the plots' bounding box when the vis window is in 2D mode. If
-you want to turn off the guidelines, click off the **Show Guidelines** check
-box in the **Interactors window**.
+The zoom interactor settings are mostly used when the vis window is in zoom mode. 
```

Comment:
Using `vis window` (here and a few other places). Is that really user-facing terminology? I don't think so. Maybe best to use `viewer` or just `window` (e.g. drop the `vis` part).

Reply:
I didn't write that section. I just changed to single line per sentence.

Reply:
Is there a way to capture your suggestions to a new ticket?

Reply:
> I didn't write that section. I just changed to single line per sentence.

I suppose I should have been able to pick up on that in examining the diffs. But, it hadn't occurred to me and so I am finding myself spending time looking at material you don't want reviewed.

Can you suggest anything so I can avoid this in future?

Reply:
I try to add notes to the effect that I changed to single-line per sentence. It may have been part of one of my commit messages, or I may be thinking of a different PR. At any rate, if it had been a note in my PR I think it could have been helpful. My apologies for this oversight

Reply:
> Is there a way to capture your suggestions to a new ticket?

There is something that comes close. In the elipsis in upper-right of the comment, pull down to `Reference in new issue`. That brings up a new dialog (which is pretty limited in functionality) but does allow you to create a new ticket based on the original text. I just did that for the case below :wink:

Reply:
> My apologies for this oversight

Not necessary. I was just looking for a suggested tweek to our practices.

Let me propose this...when you wanna make a pass fixing line length *globally* in a `.rst` file (except `functions.rst` :wink:) as opposed to just the paragraph you are there to edit, do that and *only* that work in a separate commit. Maybe put that commit *after* whatever *real* changes you are there to make. The first diff is the real work. The second diff can be ignored.